docs/shortfin/llm/user/e2e_llama8b_k8s.md

```diff
@@ -0,0 +1,42 @@
+# LLama 8b GPU instructions on Kubernetes
```

Comment:
I'd also keep this guide general, maybe keep it next to `llama_end_to_end.md` as `llama_serving_on_kubernetes.md`, dropping "8B" and "GPU" from the title. Could then also rename `llama_end_to_end.md` as `llama_serving.md`? IDK. Naming is hard.

I'm being picky about file names since I want to link to these guides in the release notes, which will then make renaming them later harder without creating 404s

Reply:
Cool, I think we should go with `llama_serving_on_kubernetes.md` and `llama_serving.md`. end to end can be confusing to what it entails (especially with the sglang layer on top)